Health_Log_Book/README.md

```diff
@@ -0,0 +1,14 @@
+## Health log book
+
+### Details of the script:-
+The script will contain the daily records of food and workouts of a person. The user will only enter the name of the foods, it will be stored in a text file with corresponding date and time. Similarly, the user will enter the types of workout at their workout time. The workouts will be stored in a text file with date and time. The user can also retrieve the data whenever they want to see.
+
+### Advantages of the script:-
+This script will help the user to maintain his/her health, as it is keeping a daily track of their diet and workout style. At the end of the day, or at the end of the week, he/she can analyze the data and can lead a healthy and fit lifestyle.
+
+### Modules Used:-
+
+1) datetime
+
+### Images of the script:-
+<img src = "https://user-images.githubusercontent.com/62845847/113548479-3998f980-960d-11eb-806b-4321cec05a98.png" width = "400" height = "300">
```

Comment:
This is not the complete image of the script and therefore you can remove this image. Remove this section heading too.